src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -299,12 +302,27 @@ private static void PackageManagerViewModel_PropertyChanged(object sender, Syste
                     //Put the name of the Package to be searched in the SearchTextBox
                     packageManagerViewModel.SearchText = AutodeskSamplePackage;
 
+                    searchPackagesLoaded = true;
+
+                    EnableNextButton(null, null, true, GuideFlow.FORWARD);
+
                     //Unsubscribe from the PropertyChanged event otherwise it will enter everytime the SearchTextBox is updated
-                    packageManagerViewModel.PropertyChanged -= PackageManagerViewModel_PropertyChanged;
+                    packageManagerViewModel.PropertyChanged -= searchPackagesPropertyChanged.Invoke;
+
+                    searchPackagesLoaded = false;
                 }
             }
         }
 
+        internal static void EnableNextButton(Step stepInfo, StepUIAutomation uiAutomationData, bool enableFunction, GuideFlow currentFlow)
+        {
+            if (searchPackagesLoaded)
+            {
+                var nextButton = Guide.FindChild((CurrentExecutingStep.StepUIPopup as PopupWindow).mainPopupGrid, "NextButton") as Button;
```

Comment:
Didn't we define `Next Button` somewhere already?

Reply:
Now it's defined in the JSON file. Thanks!